cmd/daemon/player.go

```diff
@@ -452,7 +447,7 @@ func (p *AppPlayer) handleApiRequest(req ApiRequest) (any, error) {
 		}
 
 		p.state.setActive(true)
-		p.state.player.PlaybackSpeed = 1
+		p.state.setPaused(data.Paused)
```

Comment:
I don't see `IsPaused` getting updated here, while I think it should be. But `PlaybackSpeed` is. I think it is more correct to update both at the same time.

Untested, though.